### ISSUE TITLE
Alpine Linux doesn't have webkit for Lightdm

### ISSUE THREAD

**Issue by Footsiefat** [grassmunk/Chicago95#298]
Since Alpine doesn't include the webkit greeter, is there any ways we could port the html login page to native somehow?

**Comment by LuisPeregrina** [comment]
Neither ubuntu 23.x

**Comment by grassmunk** [comment]
Yeah, honestly, the webkit one was the only real way to make the theme. I'm not sure how you would do it

Also @LuisPeregrina thank you for replying to these (I see you've replied to a bunch), I appreciate the help.
